Resources/config/controller.xml

```diff
@@ -9,11 +9,18 @@
             <argument type="service" id="hwi_oauth.security.oauth_utils" />
             <argument type="service" id="hwi_oauth.resource_ownermap_locator" />
             <argument type="service" id="request_stack" />
+            <argument key="$enableConnect">%hwi_oauth.connect%</argument>
+            <argument key="$grantRule">%hwi_oauth.grant_rule%</argument>
+            <argument key="$failedUseReferer">%hwi_oauth.failed_use_referer%</argument>
+            <argument key="$failedAuthPath">%hwi_oauth.failed_auth_path%</argument>
+            <argument key="$enableConnectConfirmation">%hwi_oauth.connect.confirmation%</argument>
+            <argument key="$firewallNames">%hwi_oauth.firewall_names%</argument>
 
             <call method="setContainer">
                 <argument type="service" id="service_container" />
             </call>
 
+            <tag name="container.service_subscriber" />
             <tag name="controller.service_arguments" />
```

Comment:
This can be removed too if there are no services injected as arguments in actions.